### ISSUE TITLE
Interview David Salter for MP Blog

### ISSUE THREAD

**Issue by cesarhernandezgt** [microprofile/microprofile-marketing#182]
Cesar Hernandez to interview David Salter on how involvement in MP started and create a MP blog out of the results.

Hi, @davidsalter , this is the document I created for the interview, feel free to fill out the questions you like. I think the week after code one (Septembre 23th) we can catch up with the interview process. 
https://docs.google.com/document/d/1GFwEPqVnycnC5UAjoNWP59zGdtu_uUBmTAxUn3ry89o/edit?usp=sharing

Marketing admins, please assign this ticket to David Salter since I don't have the grant's to assign tickets.

**Comment by aeiras** [comment]
to add @davidsalter, he needs to comment in this ticket :)

**Comment by davidsalter** [comment]
Thanks Cesar. I’ll go through the questions this week :)

**Comment by aeiras** [comment]
@cesarhernandezgt after all is done in the word doc, before interview gets added to wordpress, you will need to find a WordPress editor to own the final review and work with final steps. 
Do note that via forum I stated that all steps are written and that someone *not me* needs to own such steps to help with scalability on final sections. 

Edwin cannot help. 
[WordPress Editors list](https://docs.google.com/document/d/1q0Xw4XJcoVq1V5vPWe9KR__WKoGc0e-X5cD3tnBsIKw/edit)

Simply put, both of you could ask for a volunteer on the Community call too. :)

**Comment by davidsalter** [comment]
@cesarhernandezgt - my aim is to get this competed this weekend.

**Comment by davidsalter** [comment]
@cesarhernandezgt Can you take a look at the word doc and check that you think it all looks ok?

**Comment by cesarhernandezgt** [comment]
Hi @davidsalter , thanks much!
I provided feedback on the document (suggestion mode) feel free to accept/reject them.

**Comment by davidsalter** [comment]
@cesarhernandezgt - That looks fantastic to me.

**Comment by cesarhernandezgt** [comment]
@aeiras @davidsalter 
Draft post created and ready to be reviewed by a WordPress Admin volunteer &/or designer who has access to twitter management.

https://microprofile.io/?p=6420&preview=true

**Comment by aeiras** [comment]
@davidsalter & @cesarhernandezgt, 

The final review completed, I added a few links and adjusted minor things. Check it out and let me know if there are any questions. Super fun! 
personally, _I had so much fun & I appreciated learning a bit more about you David! :)_ 

@rstjames, 
Can you please own 3 steps below from  https://github.com/eclipse/microprofile-marketing/wiki/Interviewing-New-MicroProfilers?  ETA Oct 28th publishing. Please pull me in if there is an issue with push. Thanks! 

- [5 ]Designer - Social media push , already created as template

- [6]  media push

- [ 7]

**Comment by rstjames** [comment]
@davidsalter @aeiras @cesarhernandezgt 
- updated the top image for the blog post
- attached is social media image 
+ I need a tweet message to push this out 

![MicroProfile-DavidSalter-Interview-1](https://user-images.githubusercontent.com/12132146/67573060-c1e84f00-f737-11e9-8925-68fdc5ceee52.png)

**Comment by aeiras** [comment]
Please let’s use the example format used on the template wiki Ryan! 
Like that there is a constant tweet msg that enables anyone to help with this step without errors. I added the wiki link to remind you of it! Thanks

**Comment by aeiras** [comment]
![14D20EFB-C5A2-4E53-8F3A-05F54EFB9322](https://user-images.githubusercontent.com/10731224/67587305-6d79ab00-f708-11e9-8f54-757d4722e961.png)

Let’s use the template we created without deviations Ryan! Thanks

**Comment by rstjames** [comment]
tweet template: 
#MicroProfileInterview:[interviewee twitter]shares [his/her]experience about on-boarding into @MicroProfileIO with [interviewer twitter]. [interviewee's Name] provides valuable input in optimizing the welcoming process & hopefully inspires you to join as a member too. [link of interview]

**Comment by rstjames** [comment]
Updated image: @aeiras 
![MicroProfile-DavidSalter-Interview](https://user-images.githubusercontent.com/12132146/67869278-81b91000-fb2d-11e9-8d1b-beb1efc5a861.png)

**Comment by aeiras** [comment]
[Twitter push: Interview David Salter](https://twitter.com/GeefyGeorge/status/1189929061541396480?s=20)
